README.md

```diff
@@ -29,6 +44,7 @@ To build image-inspector using godep:
 
 # Running as a container
 
-    $ docker run -ti --rm --privileged \
+    $ docker run -ti --rm --privileged -p 5000:5000 \
```

Comment:
agree, the first run example in the readme (line 6) uses 8080 so I'd probably keep it consistent.